src/Item_Disk.php

```diff
@@ -713,8 +713,20 @@ public static function getSpecificValueToDisplay($field, $values, array $options
 
     public function getNonLoggedFields(): array
     {
-        return [
+        // we don't want to log at all changes of available space for a drive
+        // as it's likely to change every time
+        $exclude = [
             'freesize',
+            'date_mod',
```

Comment:
I think `date_mod` should be excluded globally and not added here.

Reply:
in fact, i'm not sure it's required at all, will check